src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/staff/StaffCommands.java

```diff
@@ -118,6 +119,8 @@ public void reload(CommandSender sender) {
         storage.importFromDatabase();
         permissions.loadPermissions();
 
+        plugin.setTagValidator(new TagValidator(settings, permissions));
```

Comment:
Just thought 🤔 
Is it even makes sense? 
Classes would be updated by their own, so don't need to create new TagValidator, right?

Reply:
It doesn't make sense since SettingsManager and PermissionManager are intended to have only one instance on runtime.